index.html

```diff
@@ -0,0 +1,252 @@
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </section>
+      <section class="card-works-1">
+        <img src="./img/Snapshoot-Portfolio-1.svg" alt="Snapshoot Portfolio" />
+        <h1 class="project-title">Multi-Post Stories</h1>
+        <ul class="frame-2">
+          <li class="client">Canopy</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="role">Back End Dev</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="year-label">2015</li>
+        </ul>
+        <p class="primary-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </section>
+      <section class="card-works">
+        <img src="./img/Snapshoot-Portfolio-2.svg" alt="Snapshoot Portfolio" />
+        <h1 class="project-title">Tonic</h1>
+        <ul class="frame-2">
+          <li class="client">Canopy</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="role">Back End Dev</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="year-label">2015</li>
+        </ul>
+        <p class="primary-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </section>
+      <section class="card-works-1">
+        <img src="./img/Snapshoot-Portfolio-3.svg" alt="Snapshoot Portfolio" />
+        <h1 class="project-title">Multi-Post Stories</h1>
+        <ul class="frame-2">
+          <li class="client">Canopy</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="role">Back End Dev</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="year-label">2015</li>
+        </ul>
+        <p class="primary-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </section>
+    </div>
+    <footer class="base-about-myself">
+      <section class="top">
```

Comment:
Even though your use of the `footer` tag is super good, kindly use a `section` tag rather than dividing the about me section into multiple other elements. This way we meet the project requirements 📝 All the about me section should be wrapped with this particular element

Reply:
I can't, I will have a linter error, because sections need heading, and I don't need one for this particular use

Reply:
- Please remove the footer tag. and try to follow the next suggestion to avoid linters errors. Implement the following structure for `About` section.
```html
<section class="base-about-myself">
   <h2>About <br />  Myself </h2>
   .....
   <ul class="skill-list">
        <li class="frame">
          <summary class="frame-summary">Languages</summary>
          <div class="language-list">
            <img src="./img/js-lang.svg" alt="" />
            <img src="./img/html-lang.svg" alt="" />
            <img src="./img/css-lang.svg" alt="" />
          </div>
        </li>
        <li class="frame">
          <summary class="frame-summary">Frameworks</summary>
        </li>
        <li class="frame">
          <summary class="frame-summary">Skills</summary>
        </li>
   </ul>
</section>
```